network-core/src/server/block.rs

```diff
@@ -64,6 +64,12 @@ pub trait BlockService {
     /// accepted by this node.
     fn tip(&mut self) -> Self::TipFuture;
 
+    /// Request to load list of blocks.
+    fn pull_blocks(&mut self, ids: &[Self::BlockId]) -> Self::PullBlocksFuture;
```

Comment:
Please rename these to `get_blocks` and `get_headers` to be consistent with the protobuf method names.